<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiFreq</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>38</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actioncopy"/>
    <addaction name="actionpaste"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionplay"/>
   <addaction name="separator"/>
   <addaction name="actionrandom"/>
   <addaction name="separator"/>
   <addaction name="actionclear"/>
   <addaction name="separator"/>
   <addaction name="actionopen_wave"/>
   <addaction name="actionsave_wave"/>
   <addaction name="separator"/>
   <addaction name="actionexport"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWave">
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">background:darkgreen; color:rgb(255,255,255)</string>
           </property>
           <property name="text">
            <string>amplitude</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="WaveDrawer" name="amp" native="true">
           <property name="accessibleName">
            <string>amp</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">background:darkgreen; color:rgb(255,255,255)</string>
           </property>
           <property name="text">
            <string>pitch (hz)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="WaveDrawer" name="freq" native="true">
           <property name="accessibleName">
            <string>freq</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">background:darkgreen; color:rgb(255,255,255)</string>
           </property>
           <property name="text">
            <string>phase</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="WaveDrawer" name="phase" native="true">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="accessibleName">
            <string>phase</string>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockTable">
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QSpinBox" name="maxRndWaves">
       <property name="toolTip">
        <string>max # of random waves</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
       <property name="value">
        <number>9</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="minFreq">
       <property name="toolTip">
        <string>min random freq</string>
       </property>
       <property name="maximum">
        <number>22000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="maxFreq">
       <property name="toolTip">
        <string>max random freq</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>22000</number>
       </property>
       <property name="value">
        <number>8000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="secs">
       <property name="toolTip">
        <string>secs. to export</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TableWave" name="tableWave">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockChords">
   <property name="windowTitle">
    <string>Chords</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="TableChords" name="tableChords">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="actioncopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/VSlibQt/rsc/icons/clipcopy.png</normaloff>:/VSlibQt/rsc/icons/clipcopy.png</iconset>
   </property>
   <property name="text">
    <string>copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionpaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/VSlibQt/rsc/icons/clippaste.ico</normaloff>:/VSlibQt/rsc/icons/clippaste.ico</iconset>
   </property>
   <property name="text">
    <string>paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionplay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/VSlibQt/rsc/icons/play.png</normaloff>
     <normalon>:/VSlibQt/rsc/icons/stop.png</normalon>:/VSlibQt/rsc/icons/play.png</iconset>
   </property>
   <property name="text">
    <string>play</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionclear">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/VSlibQt/rsc/icons/clear.png</normaloff>:/VSlibQt/rsc/icons/clear.png</iconset>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </action>
  <action name="actionrandom">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/VSlibQt/rsc/icons/rand.png</normaloff>:/VSlibQt/rsc/icons/rand.png</iconset>
   </property>
   <property name="text">
    <string>random</string>
   </property>
  </action>
  <action name="actionexport">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/VSlibQt/rsc/icons/exportsound.png</normaloff>:/VSlibQt/rsc/icons/exportsound.png</iconset>
   </property>
   <property name="text">
    <string>export</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionsave_wave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/VSlibQt/rsc/icons/save.png</normaloff>:/VSlibQt/rsc/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>save wave</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionopen_wave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/VSlibQt/rsc/icons/open.png</normaloff>:/VSlibQt/rsc/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>open wave</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WaveDrawer</class>
   <extends>QWidget</extends>
   <header location="global">wavedrawer.h</header>
   <container>1</container>
   <slots>
    <signal>dataUpdated()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>TableWave</class>
   <extends>QTableView</extends>
   <header location="global">tablewave.h</header>
   <slots>
    <slot>onDataChanged()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>TableChords</class>
   <extends>QTableView</extends>
   <header location="global">tablechords.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>amp</sender>
   <signal>dataUpdated()</signal>
   <receiver>tableWave</receiver>
   <slot>onDataChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>freq</sender>
   <signal>dataUpdated()</signal>
   <receiver>tableWave</receiver>
   <slot>onDataChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>694</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>phase</sender>
   <signal>dataUpdated()</signal>
   <receiver>tableWave</receiver>
   <slot>onDataChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>782</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
